packages/ckeditor5-markdown-gfm/docs/_snippets/features/markdown.js

```diff
@@ -9,13 +9,15 @@ import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor'
 
 import ArticlePluginSet from '@ckeditor/ckeditor5-core/tests/_utils/articlepluginset';
 import EasyImage from '@ckeditor/ckeditor5-easy-image/src/easyimage';
```

Comment:
It is a bit strange to have Easy Image here but no image feature in the toolbar. The editor itself contains the image toolbar configuration so why don't we enable it?

Reply:
Do we support markdown for images? If so, then we may. If not, then it's not really necessary, but I'd rather not tackle with the settings as of yet myself.

Reply:
It's quite easy to check it out - just drag an image to this sample and see the result. And then you can have a look at other samples with support for this feature, and finally, ping @Mgsy for a review/help with the updated settings. Sounds like a plan, no? :wink:

Reply:
I'll try to do it, though I see little purpose in that, to be honest.

Reply:
I've added the `imageUpload` button to the toolbar.

Reply:
Thanks, @Mgsy! And why so little faith, @godai78? Inserting images is something that most users expect the editor to support. If we support this with proper Markdown output (and we do), I see no reason to hide it. Does it make sense to you?